TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md

```diff
@@ -0,0 +1,119 @@
+
+* **Zoom**: [https://zoom-lfx.platform.linuxfoundation.org/meeting/94618773737?password=4b2a5cdf-685a-4ea3-822d-24ff7ddab72e) 
+
+* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io),
+[ical](https://calendar.google.com/calendar/ical/c\_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics),
+[Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c\_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com)
+
+* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+* Dan Middleton (DM) opened the call at 7:05am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Ben Sternthal (BS) created meeting minutes based on the publically available recording
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+A quorum was not required.
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* Arm - Nathaniel McCallum+  / Michael Lu
+* Google - Cfir Cohen +  / Catalin Sandu 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Alec Fernandez +
+* Red Hat - Lily Sturmann  / Yash Mankad 
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming (Broadcom)
+* Drasko Draskovic (Ultraviolet)
+* Henk Birkholz (Fraunhofer Institute)
+* Kanth Ghatraju (Oracle) 
+* Marc Meunier (Arm)
+* Mark Novak (JP Morgan Chase)
+* Matthieu Legré (Cysec)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Steveb Walli (Microsoft)
+* Sung Lee (Broadcom)
+* Thomas Fossati (Linaro)
+* Wenhui Zhang (Bytedance/TikTok)
+* Mike Bursell (CCC)
+* Sal Kimmich (LF)
```

Comment:
Doesn't Sal have the same affiliation as Mike?

Reply:
Good catch.